docs/quick.rst

```diff
@@ -94,4 +94,9 @@ Click :ref:`datasets` for a list of datasets available on SciServer.
 
 Check out `Tutorial <Tutorial.ipynb#Tutorial>`_, Use Cases, and :ref:`api` to learn more about OceanSpy and its features, and feel free to open an `issue here <https://github.com/malmans2/oceanspy/issues>`_, or to send an email to `mattia.almansi@jhu.edu <mattia.almansi@jhu.edu>`_ if you have any questions.
 
+The flowchart below is meant to be a guiding post to understand the way OceanSpy is designed to be used by the oceanographic community.
```

Comment:
Reword to:

The flowchart below shows how OceanSpy is designed to be used by the oceanographic community.

Reply:
done.